src/services/scssCompletion.ts

```diff
@@ -211,6 +211,20 @@ export class SCSSCompletion extends CSSCompletion {
 		return result;
 	}
 
+	public getCompletionsForDeclarations(declarations: nodes.Declarations, result: CompletionList): CompletionList {
+		if (!declarations || this.offset === declarations.offset) { // incomplete nodes
+			return result;
+		}
+
+		let node = declarations.findFirstChildBeforeOffset(this.offset);
```

Comment:
I think we should add that directly to cssCompletion.getCompletionsForDeclarations.